modules/end-user-guide/pages/using-a-git-provider-access-token.adoc

```diff
@@ -70,15 +70,17 @@ metadata:
     che.eclipse.org/scm-url: __<git_provider_endpoint>__# <3>
     che.eclipse.org/scm-userid: '__<git_provider_user_id>__'# <4>
```

Comment:
@max-cx 
Here is a command how to get a `scm-userid`:
`curl -s -u  :<PAT> https://vssps.dev.azure.com/<ORGANIZATION>/_apis/connectionData\?api-version\=7.0-preview.1 | jq -r '.authenticatedUser.id'`
But I don't know how to incorporate it the doc since every git provider has a different way to find out the `scm-userid`

Reply:
I've opened a Jira issue, [RHDEVDOCS-5028](https://issues.redhat.com/browse/RHDEVDOCS-5028), as part of which the docs team will document this. I suggest we skip this content in this PR and just get it merged.